### PR TITLE
Allows catgirls to have collars put on them

### PR DESCRIPTION
**Pull request by thehighwayman**
### Intent of your Pull Request
I'm told by a particular catgirl that they would like to be able to have collars put on them by other people. this will allow that while disallowing humans from being subjugated.
#### Changelog

:cl:  
tweak: lets catgirls have collars put on them 
/:cl:

### CODE REVIEW

**Comment by tsaoToast** [comment]
bad PR unless it makes them valid

**Comment by thehighwayman** [comment]
no clue why ktl likes this PR since it means I'm gonna put a collar on her every chance I get. Stockholm syndrome

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -193,9 +193,11 @@
 	var/tagname = null
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))
-		return FALSE
-
+	if(istype(M, /mob/living/carbon/human/species/felinid)) // yogs start - lets catpeople wear collars
+		return TRUE
```
Comment (mrhugo13) [inline code/modules/clothing/neck/_neck.dm]:
```suggestion
		gib()
```

**Comment by Nickvr628** [comment]
But they can already wear a collar in the next slot, like ties or scarfs.

**Comment by ShadowDeath6** [comment]
can we repo ban for this?

**Comment by Ktlwjec** [comment]
i will delete all the maps if this gets closed

File: code/modules/clothing/neck/_neck.dm [hunk 2]
```diff
@@ -193,9 +193,11 @@
 	var/tagname = null
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))
-		return FALSE
-
+	if(istype(M, /mob/living/carbon/human/species/felinid)) // yogs start - lets catpeople wear collars
```
Comment (Amelia0010) [inline code/modules/clothing/neck/_neck.dm]: you do realize we don't use this mob for purrbation, meaning this only affects people you've spawned in with this mobtype, right?

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by Amelia0010** [comment]
[22:49] Nich or miss:
if(istype(M, /mob/living/carbon/human)
    var/mob/living/carbon/human/T = M
    if(istype(T.species, /datum/species/human/felinid)
        return TRUE
    return FALSE
[22:49] Thehighwayman5 (WILDCARD): can't do that
[22:49] Thehighwayman5 (WILDCARD): you'll get an error
[22:49] Nich or miss: okay then it's not getting merged

**Review by Amelia0010 — APPROVED** [review]

File: code/modules/clothing/neck/_neck.dm [hunk 3]
```diff
@@ -193,9 +193,12 @@
 	var/tagname = null
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))
-		return FALSE
-
+	if(istype(M, /mob/living/carbon/human)  // yogs start - lets catpeople wear collars
```
Comment (Amelia0010) [inline code/modules/clothing/neck/_neck.dm]: you forgot the second parenthesis

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/neck/_neck.dm (hunk 3 above)
Comment (thehighwayman) [reply code/modules/clothing/neck/_neck.dm]: fuck you

Comment (Ktlwjec) [reply code/modules/clothing/neck/_neck.dm]: silly sausage!

**Comment by Ktlwjec** [comment]
https://forums.yogstation.net/index.php?threads/vote-merge-the-pr-to-allow-felinid-to-wear-pet-collars.17795/ yeeted

File: code/modules/clothing/neck/_neck.dm [hunk 4]
```diff
@@ -193,9 +193,12 @@
 	var/tagname = null
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))
-		return FALSE
-
+	if(istype(M, /mob/living/carbon/human))  // yogs start - lets catpeople wear collars
```
Comment (Amelia0010) [inline code/modules/clothing/neck/_neck.dm]: if(ishuman(M))

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**